.travis.yml

```diff
@@ -1,17 +1,25 @@
 sudo: required
 services:
   - docker
+addons:
+  postgresql: "9.5"
 language: python
 python:
-  - 3.5
+  - 3.6
+env:
+  - SECRET_KEY="TRAVIS-SECRET-KEY"
```

Comment:
Shouldn't this be interpolated somehow? Like `"${TRAVIS_SECRET_KEY:-default-value}"`?

Reply:
That's just a placeholder. We need to provide `SECRET_KEY`, otherwise you will get:
```
Traceback (most recent call last):
  File "manage.py", line 15, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.6/dist-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.6/dist-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.6/dist-packages/django/core/management/__init__.py", line 211, in fetch_command
    settings.INSTALLED_APPS
  File "/usr/local/lib/python3.6/dist-packages/django/conf/__init__.py", line 57, in __getattr__
    self._setup(name)
  File "/usr/local/lib/python3.6/dist-packages/django/conf/__init__.py", line 44, in _setup
    self._wrapped = Settings(settings_module)
  File "/usr/local/lib/python3.6/dist-packages/django/conf/__init__.py", line 126, in __init__
    raise ImproperlyConfigured("The SECRET_KEY setting must not be empty.")
django.core.exceptions.ImproperlyConfigured: The SECRET_KEY setting must not be empty.
```